### DISCUSSION TITLE
Stars and Accept answers exchange

### DISCUSSION THREAD

**Discussion by atiqisrak** [Schweinepriester/github-profile-achievements#216] [Q&A]
Let's exchange some stars and answer accepts.

https://github.com/atiqisrak/my-react-three/

Here is mine. Hope we help each other

**Comment by mezotv** [comment]
Hey sure sounds good. I gave you a star and if you want can mark your answer as answered on my q&a. Here is also my repo for the star https://github.com/mezotv/TodayIsThatDay

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by phamvanphuckks** [comment]
sure, I need to accept my answers

**Comment by trentmillar** [comment]
🚜 🚚 🚗 🚓 🚌 🚒

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by ganochenkodg** [comment]
Done, and pls star this https://github.com/ganochenkodg/qdrant-operator
